src/Contracts/Data/Conversion.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * This file is part of the Zest Framework.
+ *
+ * @author   Muhammad Umer Farooq <lablnet01@gmail.com>
+ * @author-profile https://www.facebook.com/Muhammadumerfarooq01/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @since 2.0.0
+ *
+ * @license MIT
+ */
+
+namespace Zest\Contracts\Data;
+
+
+interface Conversion
+{
+    /**
+     * Convert arrays to Object.
+     *
+     * @param array $array Arrays
+     *
+     * @since 2.0.0
+     *
+     * @return object
+     */
+    public static function arrayObject($array);
+
+    /**
+     * Convert Objects to arrays.
+     *
+     * @param object $object
+     *
+     * @since 2.0.0
+     *
+     * @return array
+     */
+    public static function objectArray($object);
+
+    /**
+     * Convert the bit into bytes.
+     *
+     * @param int $size The value that you want provided
+     * @param int $pre  Round the value default 2
+     *
+     * @since 3.0.0
+     *
+     * @return mixed
+     */
+    public function bitToBytes($size, $pre = 2);
+
+    /**
+     * Convert the views to relative unit.
+     *
+     * @param int $n   Views.
+     * @param int $sep Seperator.
+     *
+     * @since 3.0.0
+     *
+     * @return mixed
+     */
+    public static function views($n, $sep = ',');
+
+    /**
+     * Convert XML to arrays.
+     *
+     * @param xml object $xml xml
```

Comment:
The `xml object` annotation?

I'm not sure about what this type is defined here.

Reply:
I think mixed is best here...